src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -23,6 +23,8 @@ import com.databricks.spark.csv.util.TextFile
 
 package object csv {
 
+  val defaultCsvFormat = CSVFormat.DEFAULT.withRecordSeparator(System.getProperty("line.separator"))
```

Comment:
```getProperty``` may return null if the property is not set. Please pass it a second argument as default value.